front/src/applications/stdcm/consts.ts

```diff
@@ -9,8 +9,6 @@ export const STDCM_REQUEST_STATUS = Object.freeze({
 
 export const STDCM_TRAIN_ID = -10;
 
-export const RUNTIME_CAP = 43200;
```

Comment:
what was this?

Reply:
This corresponded to 12 hours of ‘maximum_runtime’ in seconds :)